src/wechaty_puppet_service/utils.py

```diff
@@ -40,3 +42,21 @@ def extract_host_and_port(url: str) -> Tuple[str, Optional[int]]:
         return host, int(port)
 
     return url, None
+
+
+def test_endpoint(end_point: str) -> float:
+    """
+        test_endpoint:
+        1.If there is port: telnet
+        2.If there is host/domain: ping or
+    """
+    tn = Telnet()
+    host, port = extract_host_and_port(end_point)
+    if port is None:
+        return ping(host)
+    else:
+        try:
+            tn.open(host, port=port)
+        except Exception:
+            return False
+    return True
```

Comment:
Your return data type is Wrong:
You define the return type is `float`, but you return float/bool data type. This code will not pass CI testing.

Reply:
no, there are 2 pylint errors:
1./puppet.py:37:0: W0611: Unused ping imported from ping3 (unused-import)
2./utils.py:55:4: R1705: Unnecessary "else" after "return" (no-else-return)

Reply:
i just fix these pylint errors, let it run CI test.